dali/python/nvidia/dali/backend.py

```diff
@@ -28,11 +29,13 @@
     Init(OpSpec("CPUAllocator"), OpSpec("PinnedCPUAllocator"), OpSpec("GPUAllocator"))
     initialized = True
 
-    # pybind11 deprecations
-    def asCPU(self):
-        warnings.warn("asCPU is deprecated since v0.7, please use as_cpu", DeprecationWarning, stacklevel=2)
-        return self.as_cpu()
-    TensorListGPU.asCPU = asCPU
+    # py27 deprecation
+    if sys.version_info[0] < 3:
+        warnings.simplefilter("default", append=True)
+        warnings.warn("Python 2.7 will reach the end of its life on January 1st," +
+                       "2020. Please upgrade your Python as Python 2.7 won't be " +
+                       "maintained after that date. The December release will be" +
```

Comment:
Just a thought: maybe something like `The 0.17 release will ...` will age better?

Reply:
Done